libmscore/arpeggio.cpp

```diff
@@ -22,506 +22,524 @@
+// ---------------------------------------------------------
+// draw
+// ---------------------------------------------------------
 
 void Arpeggio::draw(QPainter* p) const
-      {
-      if (_hidden)
-            return;
-      qreal _spatium = spatium();
-
-      qreal y1 = -_userLen1;
-      qreal y2 = _height + _userLen2;
-
-      p->setPen(QPen(curColor(),
-         score()->styleS(Sid::ArpeggioLineWidth).val() * _spatium,
-         Qt::SolidLine, Qt::RoundCap));
-
-      p->save();
-      switch (arpeggioType()) {
-            case ArpeggioType::NORMAL:
-            case ArpeggioType::UP:
-                  {
-                  QRectF r(symBbox(symbols));
-                  qreal scale = p->worldTransform().m11();
-                  p->rotate(-90.0);
-                  score()->scoreFont()->draw(symbols, p, magS(), QPointF(-r.right() - y1, -r.bottom() + r.height()), scale);
-                  }
-                  break;
-
-            case ArpeggioType::DOWN:
-                  {
-                  QRectF r(symBbox(symbols));
-                  qreal scale = p->worldTransform().m11();
-                  p->rotate(90.0);
-                  score()->scoreFont()->draw(symbols, p, magS(), QPointF(-r.left() + y1, -r.top() - r.height()), scale);
-                  }
-                  break;
-
-            case ArpeggioType::UP_STRAIGHT:
-                  {
-                  QRectF r(symBbox(SymId::arrowheadBlackUp));
-                  qreal x1 = _spatium * .5;
-                  drawSymbol(SymId::arrowheadBlackUp, p, QPointF(x1 - r.width() * .5, y1 - r.top()));
-                  y1 -= r.top() * .5;
-                  p->drawLine(QLineF(x1, y1, x1, y2));
-                  }
-                  break;
-
-            case ArpeggioType::DOWN_STRAIGHT:
-                  {
-                  QRectF r(symBbox(SymId::arrowheadBlackDown));
-                  qreal x1 = _spatium * .5;
-
-                  drawSymbol(SymId::arrowheadBlackDown, p, QPointF(x1 - r.width() * .5, y2 - r.bottom()));
-                  y2 += r.top() * .5;
-                  p->drawLine(QLineF(x1, y1, x1, y2));
-                  }
-                  break;
-
-            case ArpeggioType::BRACKET:
-                  {
-                  qreal w = score()->styleS(Sid::ArpeggioHookLen).val() * _spatium;
-                  p->drawLine(QLineF(0.0, y1, 0.0, y2));
-                  p->drawLine(QLineF(0.0, y1, w, y1));
-                  p->drawLine(QLineF(0.0, y2, w, y2));
-                  }
-                  break;
-            }
-      p->restore();
-      }
+{
+    if (_hidden)
+        return;
+    qreal _spatium = spatium();
+
+    qreal y1 = -_userLen1;
+    qreal y2 = _height + _userLen2;
+
+    p->setPen(QPen(curColor(),
+                   score()->styleS(Sid::ArpeggioLineWidth).val() * _spatium,
+                   Qt::SolidLine, Qt::RoundCap));
+
+    p->save();
+    switch (arpeggioType()) {
+    case ArpeggioType::NORMAL:
+    case ArpeggioType::UP:
+    {
```

Comment:
я бы присмотрелся к таким свойствам:
- indent_switch_case
- indent_case_shift

Мб получится зафорсить его на case `ArpeggioType::UP: {`

Reply:
I really don't know why we shouldn't indent the cases. The cases are in brackets, why no indentation?

Reply:
Our idea is to keep the style as close as possible to some general style so that we have fewer disputes and for a large number of people he was familiar. As a general style, we adopted the Qt style.  In Qt style and in default behaviour QtCreator cases are placed without indentation.